features/step_definitions/wait_steps.rb

```diff
@@ -116,13 +116,11 @@
 end
 
 When('I wait a variable time for elements to appear') do
-  @test_site.home.wait_for_lots_of_links
-  @test_site.home.wait_for_lots_of_links(0.1)
+  @test_site.home.wait_for_lots_of_links(2.1)
```

Comment:
On the previous code it was waiting 2.1 seconds, so this is a like for like replacement. I've not looked into this, but obviously this will terminate earlier (As this waiter will pass in ~1s)